bookmarks.js

```diff
@@ -1,3 +1,21 @@
+import { getSections } from './styles/lib/db.js'
+import { createCard } from './styles/lib/card.js'
+
+const content = document.querySelector('.container__content')
+
+const data = getSections()
+
+const filteredCards = createCard.filter((bookmark) => {
+
+  if(bookmark === 'black') {
```

Comment:
`bookmark` is just the hole object. But you want to check some properties in it, isn't it?